### PR TITLE
upd: lcd client config type

### PR DESCRIPTION
**Pull request by ericHgorski**

### CODE REVIEW

File: src/client/lcd/api/IbcAPI.ts
Comment (ericHgorski) [inline src/client/lcd/api/IbcAPI.ts:1]: I believe this a change @emidev98 had requested? feel free to discard if not.

Comment (emidev98) [reply src/client/lcd/api/IbcAPI.ts:1]: Thanks for checking, this change shouldn't be merged yet

**Review by emidev98 — COMMENTED** [review]
Some thoughts about the PR. I don't know if this is the right place to add these properties, because feather.js is both a platform agnostic JavaScript client. I would suggest to extend the `LCDClientConfig` when needed to add new properties instead of modifying the base class when for example an `icon` or `explorer` is not very useful in a backend. 

What do you think @ericHgorski ?

File: src/client/lcd/api/IbcAPI.ts
Comment (alecande11) [reply src/client/lcd/api/IbcAPI.ts:1]: Hey, why do we need this?
Icons, the chain name, and explorer urls are not used by feather.js.

**Comment by ericHgorski** [comment]
sure thing! I was extending but it felt a little counterintuitive. your reasoning makes sense though Ill close. @emidev98 @alecande11

**Pull request closed without merging**